library/core/src/sync/atomic.rs

```diff
@@ -79,7 +79,7 @@
 //!
 //! A simple spinlock:
 //!
-//! ```
+//! ```no_run
```

Comment:
Is there a reason to make this `no_run` now?

Reply:
No, I thought this won't finish running.
```suggestion
//! ```
```